python/orion_finance_sdk_py/costs/estimator.py

```
@@ -0,0 +1,221 @@
+            timestamp: UTC calendar date ``YYYY-MM-DD``. ``None`` means now.
+            netting_eta: Fraction of the nominal size that is internally
+                netted. The venue swap is ``(1 - eta) * signed_size``; cost
+                percentages are those of that swap, not scaled by ``(1-eta)``.
+            venue: Backend selector. Only ``uniswap_v3`` is implemented.
+        """
+        if venue not in _SUPPORTED_VENUES:
+            raise ValueError(
+                f"Unsupported venue {venue!r}. v1 supports uniswap_v3 only."
+            )
+        if signed_size == 0:
+            raise ValueError("signed_size must be non-zero")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)